sda-commons-cloudevents/src/main/java/org/sdase/commons/spring/boot/cloudevents/CloudEventV1.java

```diff
@@ -23,48 +23,56 @@
  *     Specification</a>
  */
 @JsonClassDescription(
-    "This is a manifestation of the [CloudEvents](https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/spec.md) "
-        + "specification. The field documentation will contain the official documentation and also "
-        + "some hints how values should should be set when used on the SDA platform.")
+    """
+    This is a manifestation of the
+    [CloudEvents](https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/spec.md)
+    specification.
+    The field documentation will contain the official documentation and also some hints how values
+    should should be set when used on the SDA platform.
+    """)
 public class CloudEventV1<T> extends BaseCloudEvent {
 
   @JsonPropertyDescription(
-      "Identifies the event. Producers MUST ensure that `source` + `id` is "
-          + "unique for each distinct event. If a duplicate event is re-sent (e.g. due to a "
-          + "network error) it MAY have the same id. Consumers MAY assume that Events with "
-          + "identical `source` and `id` are duplicates.\n"
-          + "\n"
-          + "**SDA**: The default is to use a random UUID.")
+      """
+      Identifies the event. Producers MUST ensure that `source` + `id` is unique for each distinct
+      event.
+      If a duplicate event is re-sent (e.g. due to a network error) it MAY have the same id.
+      Consumers MAY assume that Events with identical `source` and `id` are duplicates.
```

Comment:
The uppercase usage for **must** and **may** is intentional, or?

Reply:
Yes. Most of docs are copied from [the original spec](https://github.com/cloudevents/spec/blob/main/cloudevents/spec.md#notational-conventions) (maybe an older variant).